app/services/freckle_service.rb

```diff
@@ -25,9 +25,10 @@ def self.import_entries(start_date, end_date)
     result = client.get_entries(from: start_date, to: end_date)
     save_entries_for(result)
 
-    if result.try(:last_page)
-      last_page = result.last_page.match(/page=(\d+)/)[1].to_i
-      [2..last_page].each do |page|
+    if last = result.try(:link).try(:last)
+      last_page = last.match(/page=(\d+)/)[1].to_i
+      pages = (2..last_page)
```

Comment:
@mauro-oto why did you add the `pages` variable here? It's pretty small and gets just used once.